src/urbit/mar/chess/game/archived.hoon

```diff
@@ -18,7 +18,10 @@
         ['white' [%s (player-string:chess white.game)]]
         ['black' [%s (player-string:chess black.game)]]
         ['result' [%s ?~(result.game '' u.result.game)]]
+        ::  default values. updated thru [%archived-games ~] sub.
```

Comment:
This comment is inaccurate: the only default value remaining (`moves`) is updated through the `[%x %game @ta %moves ~]` scry